optaplanner-quarkus-integration/optaplanner-quarkus/runtime/pom.xml

```diff
@@ -67,7 +67,7 @@
     </dependency>
     <dependency>
       <groupId>org.kie.kogito</groupId>
-      <artifactId>kogito-api</artifactId>
+      <artifactId>kogito-legacy-api</artifactId>
```

Comment:
@mariofusco 
I removed the dependency you added to the [integration test](https://github.com/kiegroup/optaplanner/pull/1402/commits/1ae9261d3d49a55548d5bf68c1a8931eeaaec3ed) and updated the existing `kogito-api` dependency with `kogito-legacy-api`: "legacy" API is the default with `optaplanner-quarkus` so it should just bring in like it was doing with `kogito-api`